NumberLength.java

```diff
@@ -0,0 +1,96 @@
+package part2A.task1;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class NumberLength {
+    public static void main(String[] args) {
+        NumberLength numberLength = new NumberLength();
+        Author petrov = new Author("Petrov");
+        System.out.println(petrov);
+        Task getTask = new Task(23,1,2023,10,20);
+        Task taskDone = new Task(27,1,2023,22, 0);
```

Comment:
I'd suggest to rename getTask, because such naming leads to confuse, like if you're tying to get task here.
Examples: startDate and completeDate or taskReceived and taskDone.